zion/src/main/scala/edu/uci/ics/cloudberry/zion/common/Config.scala

```diff
@@ -31,5 +34,36 @@ object Config {
     FiniteDuration(split(0).toLong, split(1))
   }
 
+  object MemorySize extends Enumeration {
```

Comment:
this object should be out of the scope of `Config` or in the companion object of `Config`.